Changes

```diff
@@ -390,6 +390,10 @@ Working version
   (Antonin Décimo, review by Xavier Leroy, David Allsopp and Sébastien
   Hinderer)
 
+- #11933: Use the correct machtype when reading the code pointer from closures
+  (Nathanaëlle Courant, review by Gabriel Scherer, Stefan Muenzel and
```

Comment:
No need to list me here, I don't think I did enough to justify it.

Reply:
As you wish :)